src/unicef_security/models.py

```diff
@@ -8,12 +8,15 @@
 app_label = 'unicef_security'
 
 
-class TimeStampedModel:
-    last_modify_date = models.DateTimeField(editable=False, blank=True, auto_now_add=True,
-                                            auto_now=True)
+class TimeStampedModel(models.Model):
```

Comment:
I guess we can import this from model_utils

Reply:
sure, not sure how much it matters, but using model_utils will remove `last_modify_date` field from olc user